tensorflow/c/kernels.h

```diff
@@ -128,6 +134,15 @@ TF_CAPI_EXPORT extern void TF_DeleteKernelBuilder(TF_KernelBuilder* builder);
 // --------------------------------------------------------------------------
 // OpKernelContext routines
 
+// TF_GetStream returns the SP_Stream available in ctx.
+// This function returns a stream only for devices registered using the
+// StreamExecutor C API
+// (tensorflow/c/experimental/stream_executor/stream_executor.h). It will return
+// nullptr in all other cases.
+// Experimental: this function doesn't have compatibility guarantees and subject
+// to change at any time."
```

Comment:
Nit. Remove quote " at the end of this sentence.

Reply:
done